src/control/cmd/dmg/system.go

```diff
@@ -106,7 +106,9 @@ type systemQueryCmd struct {
 	ctlInvokerCmd
 	cmdutil.JSONOutputCmd
 	rankListCmd
-	Verbose bool `long:"verbose" short:"v" description:"Display more member details"`
+	Verbose       bool   `long:"verbose" short:"v" description:"Display more member details"`
+	NotJoinedOnly bool   `long:"--not-ok" description:"Only show engines not in a joined state"`
```

Comment:
As discussed in the ticket, the idea here is that the rest of tooling will gradually support the `--not-ok` admin assistance feature. So at a minimum, the field name should probably be `NotOK`, and the description should be more generic, e.g. "Display components in need of administrative investigation". 

Longer term, you might want to add a new embeddable struct in `common/cmdutil`, e.g. 'NotOkCmd` so that some common functionality can be easily added to commands as they are enhanced with the feature. It's probably fine to wait until the next command is updated before doing that, though, in order to better understand what's common. I still think that the wording should be kept somewhat generic now though, to avoid needing to change it later.

Reply:
done